#! /usr/bin/env python

PACKAGE = "rbx1_nav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("test_distance", double_t, 0, "Test distance in meters", 1.0, 0.0, 2.0)

gen.add("speed", double_t, 0, "Robot speed in meters per second", 0.15, 0.0, 0.3)

gen.add("tolerance", double_t, 0, "Error tolerance to goal distance in meters", 0.01, 0.0, 0.1)

gen.add("odom_linear_scale_correction", double_t, 0, "Linear correction factor", 1.0, 0.0, 3.0)

gen.add("start_test", bool_t, 0, "Check to start the test", False)


exit(gen.generate(PACKAGE, "calibrate_linear", "CalibrateLinear"))
